internal/svc/list.go

```diff
@@ -4,167 +4,13 @@
-				if strings.HasPrefix(err.Error(), "unknown key") {
-					delete(pMap, i)
-					continue
-				}
-				return nil, status.Error(codes.Internal, fmt.Sprintf("could not query policy: %s", err))
-			}
-			value := getStringValue(val)
-
-			switch filter.Type {
-			case requestv1.ListPoliciesRequest_MATCH_TYPE_EXACT:
-				if value == filter.Value {
-					continue
-				}
-				delete(pMap, i)
-			case requestv1.ListPoliciesRequest_MATCH_TYPE_WILDCARD:
-				exp := filter.Value
-				if _, ok := rMap[exp]; !ok {
-					r, err := regexp.Compile(exp)
-					if err != nil {
-						return nil, status.Error(codes.InvalidArgument, fmt.Sprintf("could not compile regex: %s", err))
-					}
-					rMap[exp] = r
-				}
-
-				if rMap[exp].MatchString(value) {
-					continue
-				}
-				delete(pMap, i)
-			default:
-				return nil, status.Error(codes.InvalidArgument, "invalid filter type")
-			}
-		}
-	}
-
-	ps := make([]*policyv1.Policy, 0, len(pMap))
-	for _, p := range pMap {
-		ps = append(ps, p)
-	}
-
-	return ps, nil
-}
-
-func sortPolicies(sortOptions *requestv1.ListPoliciesRequest_SortOptions, policies []*policyv1.Policy) {
-	if sortOptions == nil {
-		return
-	}
-
-	var data sort.Interface
-
-	switch sortOptions.Column {
-	case requestv1.ListPoliciesRequest_SortOptions_COLUMN_VERSION:
-		data = versionOrder(policies)
-	default:
-		data = nameOrder(policies)
-	}
-
-	if sortOptions.Order == requestv1.ListPoliciesRequest_SortOptions_ORDER_DESCENDING {
-		sort.Sort(sort.Reverse(data))
-	} else {
-		sort.Sort(data)
-	}
-}
-
-type nameOrder []*policyv1.Policy
-
-func (s nameOrder) Len() int { return len(s) }
-
-func (s nameOrder) Swap(i, j int) { s[i], s[j] = s[j], s[i] }
-
-func (s nameOrder) Less(i, j int) bool {
-	in, jn := getPolicyName(s[i]), getPolicyName(s[j])
-	if in == jn {
-		return getPolicyVersion(s[i]) < getPolicyVersion(s[j])
-	}
-
-	return in < jn
-}
-
-type versionOrder []*policyv1.Policy
-
-func (s versionOrder) Len() int { return len(s) }
-
-func (s versionOrder) Swap(i, j int) { s[i], s[j] = s[j], s[i] }
-
-func (s versionOrder) Less(i, j int) bool {
-	iv, jv := getPolicyVersion(s[i]), getPolicyVersion(s[j])
-	if iv == jv {
-		return getPolicyName(s[i]) < getPolicyName(s[j])
-	}
-
-	return iv < jv
-}
-
-func getPolicyName(p *policyv1.Policy) string {
-	switch pt := p.PolicyType.(type) {
-	case *policyv1.Policy_ResourcePolicy:
-		return pt.ResourcePolicy.Resource
-	case *policyv1.Policy_PrincipalPolicy:
-		return pt.PrincipalPolicy.Principal
-	case *policyv1.Policy_DerivedRoles:
-		return pt.DerivedRoles.Name
-	default:
-		return "-"
-	}
-}
-
-func getPolicyVersion(p *policyv1.Policy) string {
-	switch pt := p.PolicyType.(type) {
-	case *policyv1.Policy_ResourcePolicy:
-		return pt.ResourcePolicy.Version
-	case *policyv1.Policy_PrincipalPolicy:
-		return pt.PrincipalPolicy.Version
-	default:
-		return "-"
-	}
-}
-
 func sortSchemas(schemasIds []string) {
 	sort.Strings(schemasIds)
 }
+
+func sortPolicies(ids []string) {
+	sort.Strings(ids)
+}
```

Comment:
I think we can simply inline these two functions and get rid of `list.go` completely.